BrownField/Networking/Concepts/Connectivity-Multi-region-with-vWAN/Dual-Region-with-vWAN.md

```diff
@@ -0,0 +1,122 @@
+
+The diagram also depicts how all routes in each AVS Cloud region will learn routes from both the local and cross-regional hub. All blue routes are from Region 1, and all red routes are from Region 2. 
+
+![image](https://github.com/jasonamedina/Enterprise-Scale-for-AVS/assets/97964083/43d9ac83-d982-4cd1-8760-ff32b5dd6f76)
+
+**Traffic Flow**
+| From |   To |  Hub 1 VNets | On-Premise | Hub 2 VNets | Cross-Regional AVS Private Cloud|
+| -------------- | -------- | ---------- | ---| ---| ---|
+| AVS Cloud Region 1    | &#8594;| Hub1Fw>Vnet1|  GlobalReach(A)>OnPremise   | Hub2Fw>Vnet2 | Global Reach(C)>AVS Cloud Region 2|
+| AVS Cloud Region 2   | &#8594;|  Hub1Fw>Vnet1 |  GlobalReach(B)>OnPremise   | Hub2Fw>Vnet2 | Global Reach(C)>AVS Cloud Region 1|
+
+### On-Premise connectivity & traffic flow
+
+This section will focus only on the on-premise site. As shown in the diagram below, the On-Premise site will have an ExpressRoute connection to both Region 1 and Region 2 hubs (black connections).
+
+On-Premise can communicate to AVS Cloud Region 1 via orange connection "Global Reach (A)". On-Premise will also be able to communicate with AVS Cloud Region 2 via green connection "Global Reach (B).
+
+The diagram shows how On-Premise will learn routes from both regional hubs and both AVS Private clouds. All blue routes are from Region 1, and all red routes are from Region 2. Black routes are on-premise routes and are advertised back to Azure.
+
+![image](https://github.com/jasonamedina/Enterprise-Scale-for-AVS/assets/97964083/9ce15daf-f6a8-412a-8d5f-a01c68bd3df1)
+
+**Traffic Flow**
+| From |   To |  Hub 1 VNets | Hub 2 VNets | AVS Region 1| AVS Region 2| 
+| -------------- | -------- | ---------- | ---| ---| ---|
+| On-Premise    | &#8594;| Hub1Fw>Vnet1|  Hub2Fw>Vnet2  | Global Reach(A)>AVS Cloud Region 1 | Global Reach(B)>AVS Cloud Region 2| 
+
+### Azure Virtual Network connectivity & traffic flow
+
+This section will focus only on connectivity from an Azure Vnet perspective. As shown in the diagram below, both Vnet1 and Vnet2 will have a vnet peering directly to its local regional hub. 
+
+The diagram shows how all Azure native resources in Vnet1 and Vnet2 will learn routes under their "Effective Routes". A Secure Hub with Routing Intent enabled will always send the default RFC 1918 addresses (10.0.0.0/8, 172.16.0.0/12, 192.168.0.0/16) to peered Vnets. In our case, since Routing Intent is enabled, all resources within Vnet1 and Vnet2 will have the default RFC 1918 address with a next-hop of their local regional hub firewall. All traffic ingressing and egressing the Vnets will always transit the Hub Firewalls. Please see the traffic flow below for more detailed information.
```

Comment:
RFC 1918 prefixes plus any additional prefixes that have been added as "Private Traffic Prefixes" - see https://learn.microsoft.com/en-us/azure/virtual-wan/how-to-routing-policies#azurefirewall

Reply:
@fguerri, made the following correction as shown highlighted below.
![image](https://github.com/Azure/Enterprise-Scale-for-AVS/assets/97964083/fd259ca8-3d75-4fea-88cc-b2a3689f31c2)